kclvm/sema/src/resolver/doc.rs

```diff
@@ -0,0 +1,599 @@
+use regex::Regex;
+use std::collections::HashSet;
+use std::fs::File;
+use std::io::prelude::*;
+use std::iter::Iterator;
+use std::path::PathBuf;
+
+// strip leading and trailing triple quotes in the original docstring content
+fn strip_quotes(original: &mut String) {
+    let quote = original.chars().next().unwrap();
```

Comment:
U can use `Result<()>` as the function return type to prevent `unwrap()` calling.